Makefile

```diff
@@ -37,13 +40,22 @@ help:
 	@echo "  doctest    to run all doctests embedded in the documentation (if enabled)"
 	@echo "  check      to run a check for frequent markup errors"
 
-clean:
+clean: clean-venv
 	-rm -rf $(BUILDDIR)/*
 
+clean-venv:
```

Comment:
Should we include `clean-venv` in the help?

Reply:
It isn't included in CPython, and I think that's because:
1. if you try to recreate the venv, it already tells you to use `make clean-venv`
2. if you run `make clean` it also cleans the venv

Because of these reasons and for consistency with CPython, I decided to leave it out.

Reply:
Yep, sounds good.